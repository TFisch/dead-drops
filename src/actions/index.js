export const getProfileImage = (userInfo) => ({
  type: 'GET_IMAGE',
  userInfo
});

export const getToken = (token) => ({
  type: 'GET_TOKEN',
  token
});