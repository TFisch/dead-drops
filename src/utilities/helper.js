export const cleanImage = (imageResponse) => {
  const image = imageResponse.data.image;
  const username = imageResponse.data.username
  console.log(username);
  return image['60x60'].url;
}

