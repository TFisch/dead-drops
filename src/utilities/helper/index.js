export const cleanUserImage = (imageResponse) => {
  const username = imageResponse.data.username;
  const imageUrl = imageResponse.data.image;
  const image = imageUrl['60x60'].url;
  let userProfile = {
    username,
    image
  };
  return userProfile;
};

export const cleanLocationImage = (imageResponse) => {
  const imageUrl = URL.createObjectURL(imageResponse);
  return imageUrl;
};
