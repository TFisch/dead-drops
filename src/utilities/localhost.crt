-----BEGIN CERTIFICATE-----
MIIC5TCCAc2gAwIBAgIJAIF82mfOglyGMA0GCSqGSIb3DQEBCwUAMBQxEjAQBgNV
BAMMCWxvY2FsaG9zdDAeFw0xODA5MTExMjQ4NTlaFw0xODEwMTExMjQ4NTlaMBQx
EjAQBgNVBAMMCWxvY2FsaG9zdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoC
ggEBAKNuDWdX4AVKpjX1JxyGObS7dQ/mqV8KHxfBOPkhb48/rtHBfCw+FLySRscX
W8N52yqrknqgMv8ZkT2+Bql6cCR8qlELz3anSXsPEdraMadZqKYiDk6zQ/nGVBM2
E4m9tCUQ4sIkNTYffubb7t+9NCse6gyqM/r0vlTwq18+a6M5Mz1yqk2iRbisKGdB
4f/N2xL8p/yFAStCT/IZLvDqHuUlQa/zEPpcbWAAZMQL3YNVlaBA7PitTRkXgXF/
lLizBYTUryjPSwcSEvW3rHJ9vZl0sctuOIaYgJumCYouKr/ZJAFa7rIdPAHj8YMD
CVm4xqDfsAcAJdd65zZBksF2XKcCAwEAAaM6MDgwFAYDVR0RBA0wC4IJbG9jYWxo
b3N0MAsGA1UdDwQEAwIHgDATBgNVHSUEDDAKBggrBgEFBQcDATANBgkqhkiG9w0B
AQsFAAOCAQEAAQ4ceuEFYNMwv0u9cV9Bm/gaBqmF0jS4FXvqRa7Rtwg9mtE26Xym
mDmuOl6BNP6nRgvIC65UOLjtS7bQGWDBvX9mJA69ur/qjcE9AwbZH8b71ixJyuoG
XXwDOL7bR3d+Y17wkiy843jHRhDTZhhg7yaehfSINx4FvbbZbn0s9IxkjcSgO1+d
zpfQ+SUf6yHiA2beMo0yPVt/Xp8srNlbk4tUdX63HufM0tgcdcy30Fuz4JGdZ/pJ
VJ3eBgZspVFavmf/MuWCdAuNW8fKQbaWapX6p+YM7GGDle+LB/BvnaHzZG4bwtjG
fWgW1cJ+MEvrv4QxPvoMAcssemFHC6gNkg==
-----END CERTIFICATE-----
