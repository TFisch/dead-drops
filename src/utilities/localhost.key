-----BEGIN PRIVATE KEY-----
MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCjbg1nV+AFSqY1
9Scchjm0u3UP5qlfCh8XwTj5IW+PP67RwXwsPhS8kkbHF1vDedsqq5J6oDL/GZE9
vgapenAkfKpRC892p0l7DxHa2jGnWaimIg5Os0P5xlQTNhOJvbQlEOLCJDU2H37m
2+7fvTQrHuoMqjP69L5U8KtfPmujOTM9cqpNokW4rChnQeH/zdsS/Kf8hQErQk/y
GS7w6h7lJUGv8xD6XG1gAGTEC92DVZWgQOz4rU0ZF4Fxf5S4swWE1K8oz0sHEhL1
t6xyfb2ZdLHLbjiGmICbpgmKLiq/2SQBWu6yHTwB4/GDAwlZuMag37AHACXXeuc2
QZLBdlynAgMBAAECggEBAILrYmlrB9a1WjjXTqGXFRsnIDvAt+WeohHRg+ZvVoMW
rODzcpmukcYOGAzdNXF+aIfiMdFyC+g3fxZK5+NUZgmUpIuQPn36vO6Lv+3l5Oj+
9g9AMN3y1X1911QcecJZ6bnXsQ99GfNlf69IeC9bzjLM7QmAP2vGK8K3mVEPHAMA
Q8kfEkMKdwFacWOMP+ZXNYg/TSC50MyOmLKfWqbUjHm1vQghYevpl9N6iYLXhI1w
SevLQTP6boDS0LvBV4Y7rggai4ssUjpcme09gaZckkn/T78dfZErfbZdwufpuFJx
2QeeyQ2j4hnReTvi8tpmvwbiDRuKWWtgNMHlhUEH8aECgYEAzK7BgJaJdvorxN4h
ak7kii571LJA7K8xMRgCmVp5hh1/00fJhgj3B24mIwbo9bhPaDlIhx7fjck2FL05
X0Qz9OKhEIf2sVpJdg0enyMKuVo8UELpvEfs11VL0IbcSLbywi0PAx5Z17xLnKkf
hs4dTWuNuosYrVR6JwT18OjCagsCgYEAzGeNMv07hB6sJOVqVbN9iIXXsV8GLxjZ
XqphGxZMJVH2k8TWTlDa0XMjWxaZhFBpwJWyrtdjC05kS0KwGopic0Av+VYpu7eY
UVaRh1Ay+rJUXKAwtYRvFbtTNtv5x172v8vLG4SQgUlhjBT8vFnSyDHS07Gpdd7D
0TXAPF7K1VUCgYAaRz10ot8XNVs1NUeY/HP/AaTIU/Jd8MgiA3Eh+ymq/FKS756B
5NjCTXP7/0EdewJG4UPchSwyaejVlM2u23TINehz1ODpOCxCScDqsXcJAPrJWy3j
DGDt7HeFWCsj8skMAPjyb40dU4FppZT0m7XLYMmbb4xFhJkCNXLV8Sw0rQKBgB7u
KHqnwSLL9Wab5IxVXdCT2EC5zVjBHNtrUshjhmA7V+Gibn9pUsNQ9e+pqs9Pz8DZ
pjE43H7wfNK6o6VQfXdNJjVHqYcDUCQ2wkzEgQJYi9V8VAKTqK+UKhJqFRDrONFr
CDAAzlESBpXI8zmZKgErMOjhwG/apqz4wj8gZPcFAoGBAKPajj9EoP6IfwV9gbtH
s+xS+so0y26aKDD9dwIGbOAKLI1uJ//sr6L1hFMmo/OcGPhIe5ZBii5o/ergncdK
DEldhr6mHXkh9Z85zMQloby4J03VM3zBKB/eVVz7To/w+n7cpyryL72tMVpxCJl/
hrT+RVxISjQq85ixfsgNSxtE
-----END PRIVATE KEY-----
