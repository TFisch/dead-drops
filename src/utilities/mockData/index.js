export const tokenMock = 'AotyU8WH1d1jf6hbf_a5y4HeZ-StFVPaN5kRpsdFOEBtoWBGwAn_ADAAAabaRTpF9OxgaLYAAAAA'

export const userReducerMock = { username: 'clark', image: 'https://via.placeholder.com/350x150' }

export const mockUsername = "Harvey";

export const mockImage = 'https://via.placeholder.com/350x150'; 