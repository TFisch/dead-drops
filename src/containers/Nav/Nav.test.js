import React from 'react';
import { shallow, mount } from 'enzyme'
import { Nav } from './'
import { mockUser, mockPropsMethod } from '../../utilities/mockData'
import { setFormActive, resetConfirm, resetFormActive } from '../../actions'
import { handleSubmitButton } from './';

describe('Nav', () => {
  let wrapper;
  const mockPropsMethodTwo = mockPropsMethod;
  const mockPropsMethodThree = mockPropsMethod;

  beforeEach(() => {
    wrapper = shallow(
      <Nav
        user={mockUser}
        setFormActive={mockPropsMethod}
        resetConfirm={mockPropsMethodTwo}
        resetFormActive={mockPropsMethodThree}
      />
    )
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  describe('handleSubmit', () => {

    it('should call setFormActive when called', () => {
      wrapper.instance().handleSubmitButton();

      expect(mockPropsMethod).toHaveBeenCalled();
    });

  });

  describe('handleProfileButton', () => {
    it('should call resetConfirm when called', () => {

      wrapper.instance().handleProfileButton();

      expect(mockPropsMethodTwo).toHaveBeenCalled();

    })

    it('should call resetFormActive when called', () => {

      wrapper.instance().handleProfileButton();

      expect(mockPropsMethodThree).toHaveBeenCalled();

    })
  });

});