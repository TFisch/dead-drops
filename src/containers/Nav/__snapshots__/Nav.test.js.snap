// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Nav
    resetConfirm={[Function]}
    resetFormActive={[Function]}
    setFormActive={[Function]}
    user={
        Object {
            "image": "https://via.placeholder.com/350x150",
            "username": "Harvey",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="wrap-left"
>
          <img
                    alt="treasure-icon"
                    className="icon"
                    src="treasure.svg"
          />
          <h1
                    className="logo"
          >
                    DEAD DROPS
          </h1>
</div>,
        <div
          className="buttons"
>
          <div
                    className="nav-buttons-wrap"
          >
                    <Link
                              replace={false}
                              to="/profile"
                    >
                              <button
                                        className="nav-button"
                                        onClick={[Function]}
                              >
                                        SUBMIT DROP
                              </button>
                    </Link>
                    <Link
                              replace={false}
                              to="/profile"
                    >
                              <button
                                        className="nav-button"
                                        onClick={[Function]}
                              >
                                        PROFILE
                              </button>
                    </Link>
                    <Link
                              replace={false}
                              to="/"
                    >
                              <button
                                        className="nav-button"
                              >
                                        LOG OUT
                              </button>
                    </Link>
          </div>
          <a
                    href="https://www.pinterest.com/deaddrops/dead-drops-official/"
          >
                    <img
                              alt="pinterest-icon"
                              src="//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png"
                    />
          </a>
</div>,
      ],
      "className": "nav",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="treasure-icon"
              className="icon"
              src="treasure.svg"
/>,
            <h1
              className="logo"
>
              DEAD DROPS
</h1>,
          ],
          "className": "wrap-left",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "treasure-icon",
              "className": "icon",
              "src": "treasure.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "DEAD DROPS",
              "className": "logo",
            },
            "ref": null,
            "rendered": "DEAD DROPS",
            "type": "h1",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="nav-buttons-wrap"
>
              <Link
                            replace={false}
                            to="/profile"
              >
                            <button
                                          className="nav-button"
                                          onClick={[Function]}
                            >
                                          SUBMIT DROP
                            </button>
              </Link>
              <Link
                            replace={false}
                            to="/profile"
              >
                            <button
                                          className="nav-button"
                                          onClick={[Function]}
                            >
                                          PROFILE
                            </button>
              </Link>
              <Link
                            replace={false}
                            to="/"
              >
                            <button
                                          className="nav-button"
                            >
                                          LOG OUT
                            </button>
              </Link>
</div>,
            <a
              href="https://www.pinterest.com/deaddrops/dead-drops-official/"
>
              <img
                            alt="pinterest-icon"
                            src="//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png"
              />
</a>,
          ],
          "className": "buttons",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Link
                  replace={false}
                  to="/profile"
>
                  <button
                                    className="nav-button"
                                    onClick={[Function]}
                  >
                                    SUBMIT DROP
                  </button>
</Link>,
                <Link
                  replace={false}
                  to="/profile"
>
                  <button
                                    className="nav-button"
                                    onClick={[Function]}
                  >
                                    PROFILE
                  </button>
</Link>,
                <Link
                  replace={false}
                  to="/"
>
                  <button
                                    className="nav-button"
                  >
                                    LOG OUT
                  </button>
</Link>,
              ],
              "className": "nav-buttons-wrap",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <button
                    className="nav-button"
                    onClick={[Function]}
>
                    SUBMIT DROP
</button>,
                  "replace": false,
                  "to": "/profile",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "SUBMIT DROP",
                    "className": "nav-button",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "SUBMIT DROP",
                  "type": "button",
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <button
                    className="nav-button"
                    onClick={[Function]}
>
                    PROFILE
</button>,
                  "replace": false,
                  "to": "/profile",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "PROFILE",
                    "className": "nav-button",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "PROFILE",
                  "type": "button",
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": <button
                    className="nav-button"
>
                    LOG OUT
</button>,
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "LOG OUT",
                    "className": "nav-button",
                  },
                  "ref": null,
                  "rendered": "LOG OUT",
                  "type": "button",
                },
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="pinterest-icon"
                src="//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png"
/>,
              "href": "https://www.pinterest.com/deaddrops/dead-drops-official/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "pinterest-icon",
                "src": "//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="wrap-left"
>
            <img
                        alt="treasure-icon"
                        className="icon"
                        src="treasure.svg"
            />
            <h1
                        className="logo"
            >
                        DEAD DROPS
            </h1>
</div>,
          <div
            className="buttons"
>
            <div
                        className="nav-buttons-wrap"
            >
                        <Link
                                    replace={false}
                                    to="/profile"
                        >
                                    <button
                                                className="nav-button"
                                                onClick={[Function]}
                                    >
                                                SUBMIT DROP
                                    </button>
                        </Link>
                        <Link
                                    replace={false}
                                    to="/profile"
                        >
                                    <button
                                                className="nav-button"
                                                onClick={[Function]}
                                    >
                                                PROFILE
                                    </button>
                        </Link>
                        <Link
                                    replace={false}
                                    to="/"
                        >
                                    <button
                                                className="nav-button"
                                    >
                                                LOG OUT
                                    </button>
                        </Link>
            </div>
            <a
                        href="https://www.pinterest.com/deaddrops/dead-drops-official/"
            >
                        <img
                                    alt="pinterest-icon"
                                    src="//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png"
                        />
            </a>
</div>,
        ],
        "className": "nav",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="treasure-icon"
                className="icon"
                src="treasure.svg"
/>,
              <h1
                className="logo"
>
                DEAD DROPS
</h1>,
            ],
            "className": "wrap-left",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "treasure-icon",
                "className": "icon",
                "src": "treasure.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "DEAD DROPS",
                "className": "logo",
              },
              "ref": null,
              "rendered": "DEAD DROPS",
              "type": "h1",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="nav-buttons-wrap"
>
                <Link
                                replace={false}
                                to="/profile"
                >
                                <button
                                                className="nav-button"
                                                onClick={[Function]}
                                >
                                                SUBMIT DROP
                                </button>
                </Link>
                <Link
                                replace={false}
                                to="/profile"
                >
                                <button
                                                className="nav-button"
                                                onClick={[Function]}
                                >
                                                PROFILE
                                </button>
                </Link>
                <Link
                                replace={false}
                                to="/"
                >
                                <button
                                                className="nav-button"
                                >
                                                LOG OUT
                                </button>
                </Link>
</div>,
              <a
                href="https://www.pinterest.com/deaddrops/dead-drops-official/"
>
                <img
                                alt="pinterest-icon"
                                src="//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png"
                />
</a>,
            ],
            "className": "buttons",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Link
                    replace={false}
                    to="/profile"
>
                    <button
                                        className="nav-button"
                                        onClick={[Function]}
                    >
                                        SUBMIT DROP
                    </button>
</Link>,
                  <Link
                    replace={false}
                    to="/profile"
>
                    <button
                                        className="nav-button"
                                        onClick={[Function]}
                    >
                                        PROFILE
                    </button>
</Link>,
                  <Link
                    replace={false}
                    to="/"
>
                    <button
                                        className="nav-button"
                    >
                                        LOG OUT
                    </button>
</Link>,
                ],
                "className": "nav-buttons-wrap",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <button
                      className="nav-button"
                      onClick={[Function]}
>
                      SUBMIT DROP
</button>,
                    "replace": false,
                    "to": "/profile",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "SUBMIT DROP",
                      "className": "nav-button",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "SUBMIT DROP",
                    "type": "button",
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <button
                      className="nav-button"
                      onClick={[Function]}
>
                      PROFILE
</button>,
                    "replace": false,
                    "to": "/profile",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "PROFILE",
                      "className": "nav-button",
                      "onClick": [Function],
                    },
                    "ref": null,
                    "rendered": "PROFILE",
                    "type": "button",
                  },
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": <button
                      className="nav-button"
>
                      LOG OUT
</button>,
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "LOG OUT",
                      "className": "nav-button",
                    },
                    "ref": null,
                    "rendered": "LOG OUT",
                    "type": "button",
                  },
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="pinterest-icon"
                  src="//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png"
/>,
                "href": "https://www.pinterest.com/deaddrops/dead-drops-official/",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "pinterest-icon",
                  "src": "//assets.pinterest.com/images/pidgets/pinit_fg_en_round_red_32.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "a",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "disableLifecycleMethods": true,
    "hydrateIn": undefined,
  },
}
`;
